shield_functions.lua

```diff
@@ -3,10 +3,13 @@ castle_shields.register_shield = function(name, desc, background_color, foregrou
 	local tile_side = "castle_shield_"..background_color..".png"
 	local tile_front = "castle_shield_"..background_color..".png^(castle_shield_"..foreground_color..
 			".png^[mask:castle_shield_mask_"..mask..".png)"
+	local wield_front = "castle_shield_"..background_color..".png^(castle_shield_"..foreground_color..
+		".png^[mask:castle_shield_mask_"..mask..".png)^[mask:castle_shield_overlay.png"
```

Comment:
1. How about this?
```suggestion
	local wield_front = tile_front .. "^[mask:castle_shield_overlay.png"
```

2. Would you please be so nice to `optipng -strip all -o7` or `pngcrush` the new texture?